### PR TITLE
Replace Log_OC with timber

### PR DESCRIPTION
**Pull request by abelgardep**
Implements #2757 
Needs https://github.com/owncloud/android-library/pull/293

### CODE REVIEW

**Comment by abelgardep** [comment]
Ready to code review @davigonz. Could be a good moment to update some log types or even delete some unnecessary.

**Comment by hannesa2** [comment]
If you are done, I recommend to squash the commits

**Comment by davigonz** [comment]
@abelgardep could you check the Bitrise build? Lint is failing

File: owncloudApp/src/main/java/com/owncloud/android/datamodel/UploadsStorageManager.java [hunk 1]
```diff
@@ -196,16 +191,16 @@ private int updateUploadInternal(Cursor c, UploadStatus status, UploadResult res
...
+     * @param result     new result of upload operation
      * @param remotePath path of the file to upload in the ownCloud storage
-     * @param localPath path of the file to upload in the device storage
+     * @param localPath  path of the file to upload in the device storage
      * @return 1 if file status was updated, else 0.
      */
     public int updateUploadStatus(long id, UploadStatus status, UploadResult result, String remotePath,
                                   String localPath) {
-        //Log_OC.v(TAG, "Updating "+filepath+" with uploadStatus="+status +" and result="+result);
+        //Timber.v("Updating "+filepath+" with uploadStatus="+status +" and result="+result);
```
Comment (davigonz) [inline owncloudApp/src/main/java/com/owncloud/android/datamodel/UploadsStorageManager.java]: Same

File: owncloudApp/src/main/java/com/owncloud/android/services/OperationsService.java [hunk 2]
```diff
@@ -349,7 +347,7 @@ public boolean dispatchResultIfFinished(int operationId,
             if (undispatched != null) {
                 listener.onRemoteOperationFinish(undispatched.first, undispatched.second);
                 return true;
-                //Log_OC.e(TAG, "Sending callback later");
+                //Timber.e("Sending callback later");
```
Comment (davigonz) [inline owncloudApp/src/main/java/com/owncloud/android/services/OperationsService.java]: Same

File: owncloudApp/src/main/java/com/owncloud/android/services/OperationsService.java [hunk 3]
```diff
@@ -410,7 +408,7 @@ public void handleMessage(Message msg) {
          */
         private void nextOperation() {
 
-            //Log_OC.e(TAG, "nextOperation init" );
+            //Timber.e("nextOperation init" );
```
Comment (davigonz) [inline owncloudApp/src/main/java/com/owncloud/android/services/OperationsService.java]: Same

**Comment by davigonz** [comment]
@abelgardep a few minor changes

**Comment by davigonz** [comment]
@jesmrec code approved, ready to test

**Comment by jesmrec** [comment]
This one is a difficult to test. Checking logs (app and Logcat) in basic operations: login, upload, download, remove, create folder, sharing.

We will have to iterate and improve when more or different logs are needed. From my side it is ok to move.

**Comment by hannesa2** [comment]
I didn't followed this pull request, but my expectation was a change of these lines
https://github.com/owncloud/android/blob/0596b883f249f80a590a186f1b84b540196c3e3a/owncloudApp/src/main/java/com/owncloud/android/MainApp.kt#L264
to 
`Timber.e(...)` as well

I found 25 occurrences

**Comment by jesmrec** [comment]
thanks for your feedback @hannesa2. It is not a problem to iterate and make this changes in another issue/PR. What do you think @abelgardep?

**Pull request merged**